app/models/assignment.rb

```diff
@@ -675,17 +675,23 @@ def summary_csv(role)
     if role.instructor?
       groupings = self.groupings
                       .includes(:group,
-                                :accepted_students,
                                 current_result: :marks)
+
     else
       groupings = self.groupings
                       .includes(:group,
-                                :accepted_students,
                                 current_result: :marks)
                       .joins(:memberships)
                       .where('memberships.role_id': role.id)
     end
 
+    students = Student.all
+                      .includes(accepted_groupings: :assignment)
```

Comment:
It's sufficient to just include `accepted_groupings`, but not `:assignment`, since the `Grouping` model has an `assessment_id` attribute that you can access directly, without actually loading the associated `Assignment`